app/src/main/java/org/simple/clinic/searchresultsview/PatientSearchViewController.kt

```diff
@@ -50,10 +50,9 @@ class PatientSearchViewController @Inject constructor(
 
     return searchResultsStream
         .compose(PartitionSearchResultsByVisitedFacility(bloodPressureDao, currentFacilityStream))
-        .withLatestFrom(currentFacilityStream)
-        .map { (results, currentFacility) ->
+        .map { results ->
           { ui: Ui ->
-            ui.updateSearchResults(results, currentFacility)
+            ui.updateSearchResults(results)
```

Comment:
@sanchita I'm trying to understand the decision to get rid of the `currentFacilityStream` can you please explain.

Reply:
I refactored the class `PatientSearchResults` to capture all the required data for generating the partitioned search results by facility. That was the whole purpose of that class in the first place. I added `currentFacility` in that class itself and removed it from `SearchResultsItemType.from(patientSearchResults)`. Hence this refactor.

I think it would also make sense to rename `PatientSearchResults` to `PatinetSearchResultsByFacility` or something like that.
Wdyt?